djangocms_attributes_field/fields.py

```diff
@@ -22,16 +21,31 @@
                                      flags=re.IGNORECASE, code='invalid')
 
 
-class AttributesFormField(JSONFormField):
-    """
-    Sub-classed to set the default widget to AttributesWidget.
-    """
+class AttributesFormField(forms.CharField):
+    empty_values = [None, '']
+
     def __init__(self, *args, **kwargs):
         kwargs.setdefault('widget', AttributesWidget)
         super(AttributesFormField, self).__init__(*args, **kwargs)
 
+    def to_python(self, value):
+        if isinstance(value, six.string_types) and value:
+            try:
+                return json.loads(value)
+            except ValueError as exc:
+                raise forms.ValidationError(
+                    'JSON decode error: %s' % (six.u(exc.args[0]),)
+                )
+        else:
+            return value
+
+    def validate(self, value):
+        # This is required in older django versions.
+        if value in self.empty_values and self.required:
+            raise forms.ValidationError(self.error_messages['required'], code='required')
 
-class AttributesField(jsonfield.JSONField):
+
+class AttributesField(models.Field):
     """
```

Comment:
Sorry, meant the lines below, which mention that it's a sub-class of JSONField (which is not correct anymore)